### DISCUSSION TITLE
Moving the documenration into another repository

### DISCUSSION THREAD

**Discussion by ariannargesi** [saadeghi/daisyui#2176] [Ideas / feature requests]
Thank you all for daisyui.
Would be great if you can move the documentation into another repository. I want to contribute to this project by fixing something in the documentation and because of that, I tried to download the repository and after downloading around 1GB something failed.
I believe if we can improve this part, move developer would be encouraged to contribute into this project.
if you need any help with that, Please let me know.

**Comment by saadeghi** [comment]
The document site is not the problem (`20MB` only)  

`1GB` is the Git history because it contains data and history from all branches (including the branches for previous versions and static branches for Github Pages)

If you [fork the repo using GitHub](https://github.com/saadeghi/daisyui/fork) there is a checkbox to only fork the `master` branch.  
Then if you clone that forked repo to your local it will be something about `40MB`  

(This is something you should do for contributing to every other repo as well because usually you don't need all branches)

<img width="734" alt="Screenshot 2023-07-27 at 19 14 55" src="https://github.com/saadeghi/daisyui/assets/7342023/c0fbd81b-68e2-4e13-a527-ad0b90371482">

This guide may be helpful:  
https://github.com/saadeghi/daisyui/blob/master/.github/CONTRIBUTING.md#building-on-local

Let me know if you have any questions
